cmd/create/cluster/cmd.go

```diff
@@ -493,13 +517,53 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Autoscaling
+	autoscaling := args.autoscalingEnabled
+	if !autoscaling && interactive.Enabled() {
+		autoscaling, err = interactive.GetBool(interactive.Input{
+			Question: "Enable autoscaling",
+			Help:     cmd.Flags().Lookup("enable-autoscaling").Usage,
+			Default:  autoscaling,
+			Required: false,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid value for enable-autoscaling: %s", err)
+			os.Exit(1)
+		}
+	}
+
+	minReplicas := args.minReplicas
+	maxReplicas := args.maxReplicas
+	if autoscaling && interactive.Enabled() {
```

Comment:
what if min/max replicas value was provided in the command line?

Reply:
Since these are required, I would make it so that they are shown even if the interactive mode is not set. Something like
```go
if autoscaling && !cmd.Flags().Changed("min-replicas") {
...
}
```
Unless there's a default, in which case we can skip the interactive (unless the user sets `--interactive`) and just set the default.

Reply:
fixed